docs/data_exploration_cleaning_tagging_in_atlas.ipynb

```diff
@@ -172,7 +172,7 @@
     }
    ],
    "source": [
-    "tags = map.get_tags()\n",
+    "tags = map.tags.get_tags()\n",
```

Comment:
Ideally `map.tags` would do the retrieval on its own, not requiring a call to `map.tags.get_tags()`.

Reply:
Done

Reply:
Notebook needs to be updated as well?